lib/fs.js

```diff
@@ -3040,8 +3040,7 @@ ObjectDefineProperties(fs, {
     configurable: true,
     enumerable: true,
     get() {
-      if (promises === null)
-        promises = require('internal/fs/promises').exports;
+      promises ??= require('internal/fs/promises').exports;
```

Comment:
This is not technically the same, but it works since `promises` will never be `undefined`.

Reply:
Indeed it is :)